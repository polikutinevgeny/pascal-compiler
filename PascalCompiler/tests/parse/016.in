   function foo(): integer;
   var
      a: integer;
   begin
      foo();
   end;

var
   a: integer;
begin
   if foo() then
      a := 30
end.
