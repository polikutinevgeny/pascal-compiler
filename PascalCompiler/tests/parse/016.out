Expected identifier, constant or expression, got Minus at 1:1
